gapis/service/path/path.proto

```diff
@@ -333,6 +333,8 @@ message CommandTree {
   // If positive, synthetic sub-nodes are created for long spans of commands
   // between groups. This ensures the groups do not get lost in the noise.
   int32 max_neighbours = 14;
+  // If true, only shows draw calls from inside a command buffer
+  bool only_executed_draws = 15;
```

Comment:
This should go into the `CommandFilter` struct above.

Reply:
I need to look into how much refactoring is needed to make filters useful. The way it works now, the filter would pass for recording draw calls (Host Coordination), but failt for VkQueueSubmits that house the actual draw calls we want.